frontend/src/components/LoginForm.tsx

```diff
@@ -0,0 +1,155 @@
+/* @jsxImportSource preact */
+
+import "../styles/LoginForm.scss";
+import type { JSX } from "preact";
+import { useEffect, useState } from "preact/hooks";
+import { verifyPassword, verifyEmail, registerAccount, loginAccount, verifySession } from "../api/theBackend";
+import { fetchJSessionId, getLocalUntisCredentials, saveUntisCredentials,  } from "../api/untisAPI";
+import { generateKey, passwordDecrypt, passwordEncrypt } from "../api/encryption";
+
+export default function LoginForm(): JSX.Element  {
+    const [activeButton, setActiveButton] = useState<number>(1);
```

Comment:
1. Name this variable better, this is not an `activeButton`, it is login or register
2. Since this has only 2 states, it can be a boolean, so it could be named smth like `isRegister`